docs/content/installing.rst

```diff
@@ -19,6 +19,7 @@ Development Dependencies
 - sphinx
 - sphinx_rtd_theme
 - sphinx-gallery
```

Comment:
should we pull these into a dev-requirements.txt (sphinx and sphinx-gallery are not strictly necessary for discretize)